Source/FakeItEasy.Tests/Expressions/ExpressionCallRuleTests.cs

```diff
@@ -100,7 +100,7 @@ public void UsePredicateToValidateArguments_should_pass_predicate_to_call_matche
 
         private ExpressionCallRule CreateRule()
         {
-            return new ExpressionCallRule(this.callMatcher) { Applicator = x => { } };

```

Comment:
This didn't seem necessary (the tests still pass without it), and it made  the tests fail with the new behavior.